<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="grouplabel">
     <property name="text">
      <string>Groups:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="grouplist">
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string>Group Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>GID</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="show_sysgroups_checkbox">
     <property name="text">
      <string>Show System Groups</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="modifygroupbutton">
       <property name="text">
        <string>Modify...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newgroupbutton">
       <property name="text">
        <string>New...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deletegroupbutton">
       <property name="text">
        <string>Delete...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="groupmemberlabel">
     <property name="text">
      <string>Group Members:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="groupmemberlist">
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string>Login Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Real Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>UID</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
